### PR TITLE
JEMS.csl

### PR DESCRIPTION
**Pull request by burakkundakci**
Journal of ETA Maritime Science

### CODE REVIEW

File: JEMS [hunk 1]
```diff
@@ -0,0 +1,349 @@
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never">
+<info>
+<title>Journal of ETA Maritime Science</title>
+<title-short>JEMS</title-short>
+<id>https://csl.mendeley.com/styles/484150931/JEMS</id>
```
Comment (POBrien333) [inline JEMS]: change to: http://www.zotero.org/styles/journal-of-eta-maritime-science

File: JEMS [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+<link href="https://csl.mendeley.com/styles/484150931/JEMS" rel="self"/>
```
Comment (POBrien333) [inline JEMS]: also change to: http://www.zotero.org/styles/journal-of-eta-maritime-science

File: JEMS [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+<rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+<category citation-format="numeric"/>
+<category field="maritime"/>
```
Comment (POBrien333) [inline JEMS]: change to biology. 
It has to be one of these categories: http://docs.citationstyles.org/en/stable/specification.html#appendix-i-categories

File: JEMS (hunk 1 above)
Comment (POBrien333) [reply JEMS]: Also, change the filename to: journal-of-eta-maritime-science.csl
It has to match.

**Comment by rmzelle** [comment]
@burakkundakci, could you change the setting on this pull request so we can make some corrections for you? There should be a checkbox in the right-hand column (see the example screenshot below):

image

**Comment by burakkundakci** [comment]
> @burakkundakci, could you change the setting on this pull request so we can make some corrections for you? There should be a checkbox in the right-hand column (see the example screenshot below):
> 
> image

I change the setting. Thank you for your reply.

File: JEMS (hunk 1 above)
Comment (burakkundakci) [reply JEMS]: Thank you for your reply. I made the changes.

File: JEMS (hunk 2 above)
Comment (burakkundakci) [reply JEMS]: Thank you for your reply. I made the changes.

File: JEMS (hunk 3 above)
Comment (burakkundakci) [reply JEMS]: Thank you for your reply. I change it to engineering.

**Comment by rmzelle** [comment]
Thanks!

**Comment by burakkundakci** [comment]
> Thanks!

thank you for your help.

**Pull request merged**